tqec/detectors/transformer.py

```diff
@@ -19,6 +21,19 @@ def _fill_in_detectors_global_record_indices_impl(
         number_of_moments_explored: int = 1
         for operation in moment.operations:
             if isinstance(operation, cirq.CircuitOperation):
```

Comment:
Again, I think using different functions instead of `isinstance` makes it less brittle and more comprehensible, but that may not be possible in this case if the function is already being invoked this way.